### ISSUE TITLE
InvalidFileURI "cannot have a host" on Windows

### ISSUE THREAD

**Issue by emiltin** [davishmcclurg/json_schemer#87]
This code works on Mac:
```ruby
path = File.join(File.dirname(__dir__),'rsmp.json')
schema = JSONSchemer.schema( Pathname.new(path) )
```

But fails on Windows with:
```
JSONSchemer::InvalidFileURI:
  cannot have a host (use `file:///`)
# ./vendor/bundle/ruby/2.6.0/gems/json_schemer-0.2.17/lib/json_schemer.rb:42:in `block in <module:JSONSchemer>'
# ./vendor/bundle/ruby/2.6.0/gems/json_schemer-0.2.17/lib/json_schemer/cached_ref_resolver.rb:10:in `call'
# ./vendor/bundle/ruby/2.6.0/gems/json_schemer-0.2.17/lib/json_schemer.rb:57:in `schema'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp/message.rb:16:in `load_schemas'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp/message.rb:30:in `<class:Message>'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp/message.rb:3:in `<module:RSMP>'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp/message.rb:2:in `<top (required)>'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp.rb:30:in `require'
# ./vendor/bundle/ruby/2.6.0/bundler/gems/rsmp-9383b3296fb7/lib/rsmp.rb:30:in `<top (required)>'
# ./spec/support/spec_helper.rb:2:in `require'
# ./spec/support/spec_helper.rb:2:in `<top (required)>'
```

**Comment by davishmcclurg** [comment]
Thanks for the report @emiltin! Does https://github.com/davishmcclurg/json_schemer/pull/88 resolve the issue for you (specifically 106bfc27c031ea73132759f6d82c94b5676f216a)?

**Comment by emiltin** [comment]
Thank you @davishmcclurg, we will test it as soon as possible.

**Comment by emiltin** [comment]
It works for us

**Comment by davishmcclurg** [comment]
Fix released in 0.2.18.

Thanks @emiltin!
